src/spec/queryutils.spec.coffee

```diff
@@ -120,15 +120,15 @@ describe "QueryUtils", ->
     it "should return query predicte based on products provided", ->
       predicate = QueryUtils.matchBySku(products)
       expect(predicate).toEqual("masterVariant(sku in " +
-      "(\"1\", \"4\")) or " +
-      "variants(sku in (\"1\", \"4\"))")
+      "(\"1\\\"foo\\\"\", \"4\")) or " +
```

Comment:
Escaping the backslash that gets compared to the escaped string. I can replace these with single quotes to increase readability but as far as conventions go, there's almost no single quote in this file.